src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingBeekeeper.java

```diff
@@ -227,6 +234,19 @@ public int getMaximumHives()
         return (int) Math.pow(2, getBuildingLevel() - 1);
     }
 
+    /**
+     * Get a list of all the enabled flowers to use for breeding bees.
+     * 
+     * @return the flowers to use for the beekeeper.
+     */
+    public Set<Item> getAllowedFlowers()
+    {
+        return getModuleMatching(ItemListModule.class, m -> m.getId().equals(BUILDING_FLOWER_LIST)).getList().stream()
+                     .map(ItemStorage::getItemStack)
```

Comment:
Not 100% sure, but couldn't we save a lot of cycles by just having 1 map that does it to item directly?

Reply:
I would have probably used getItem, if I didn't overlook that method when making this when implementing that.
I will update that soon.